README.md

```diff
@@ -46,32 +87,36 @@ const inkeep =  Inkeep({
   organizationId: envConfig.ORGANIZATION_ID || '', // required
   organizationDisplayName: 'Inkeep',
   primaryBrandColor: 'black',
-  userId: '',
+  userId: '', // if you'd like analytics by user ID, like in cases where the user is authenticated or you have your own analytics platform
   userEmail: 'dev@inkeep.com',
   userName: 'Inkeep',
-  optOutAllAnalytics: true,
-  optOutAnalyticalCookies: true,
-  optOutFunctionalCookies: true,
+  optOutAllAnalytics: false,
+  optOutAnalyticalCookies: false,
+  optOutFunctionalCookies: false,
 });
 ```
-### Call the embed function
+
+You can then use inkeep.embed() multiple times with the same base settings
+
+### Syncing with dark mode and changing props after the initial render
+This can vary depending on the platform. You can use the `render` method to update an instance of the component with new properties, like the colorMode.
+
 ```
-inkeep.embed({
-  componentType: 'EmbeddedChat', // required, options: 'ChatButton', 'EmbeddedChat', 'SearchBar', 'CustomTrigger'
-  targetElement: document.getElementById('embedded-chat'), // required HTML element to render the widget into
-  properties: {
-    // optional component specific settings here
-    aiChatSettings: {
-      placeholder: 'Ask me anything...',
-      isChatSharingEnabled: true,
-      shareChatUrlBasePath: 'http://localhost:5175/shared-chat',
-      disclaimerSettings: {
-        isDisclaimerEnabled: true,
-      },
-    },
-    searchSettings: {
-      placeholder: 'Search...',
-    },
-  },
+colorModeToggle.addEventListener('change', (e) => { // whatever logic you use to track the color mode
+  const newColorMode = e.target.value;
+
+  const currentBaseSettings = inkeepWidget.baseSettings
```

Comment:
How do they get the baseSettings and do they need to re-spread them or does embed-js auto-do a deep merge so that restructuring existing props isn't needed?